src/App.js

```diff
@@ -1,64 +1,59 @@
+      const[alertType, setAlertType] = useState('');
 
-   render() {
-      return (
-      <div className="App">
-         <button className="btn btn-primary">Add Item</button>
-         <ul>
-            <li>
-               <div class="card" style={{width: '18rem'}}>
-                  <div class="card-body">
-                     <h5 class="card-title">Go Shopping</h5>
-                     <h6 class="card-subtitle mb-2 text-muted">January 5, 2021</h6>
-                     <p class="card-text">
-                        Some quick example text to build on the card title and make up the bulk of
-                        the card's content.
-                     </p>
-                  </div>
-               </div>
-            </li>
+      /**
+       * Creates a new alert and displays it to the user.
+       * After 3 seconds, the alarm is cleared
+       * @param {String} message - The alert message to be displayed
+       */
+      const createAlert = (message, type) => {
+         setAlertType(type);
+         setAlert(message);
+         setAlertStatus(true);
 
-            <li>
-               <div class="card" style={{width: '18rem'}}>
-                  <div class="card-body">
-                     <h5 class="card-title">Go Shopping</h5>
-                     <h6 class="card-subtitle mb-2 text-muted">January 5, 2021</h6>
-                     <p class="card-text">
-                        Some quick example text to build on the card title and make up the bulk of 
-                        the card's content.
-                     </p>
-                  </div>
-               </div>
-            </li>
+         setTimeout(() => {
+            setAlert('');
+            setAlertStatus('');
+            setAlertStatus(false);
+         }, 3000);
+      }
 
-            <li>
-               <div class="card" style={{width: '18rem'}}>
-                  <div class="card-body">
-                     <h5 class="card-title">Go Shopping</h5>
-                     <h6 class="card-subtitle mb-2 text-muted">January 5, 2021</h6>
-                     <p class="card-text">
-                        Some quick example text to build on the card title and make up the bulk of 
-                        the card's content.
-                     </p>
-                  </div>
-               </div>
-            </li>
-         </ul>
-      </div>
-    );
-  }
-}
+   /**
+    * If the alarm status is set to true, renders the alarm
+    * @returns {JSX} - The JSX of the alarm when the alarm status is true
+    */
+   const renderAlert = () => {
+      if (alertStatus) {
+         return (
+            <Alert variant={alertType} className="d-flex align-items-center">
+               { currentAlert }
+            </Alert>
+         );
+      }
+   };
 
-export default App;
+      return (
+         <div className="App">
+            { renderAlert() }
+            <ManageTodolist createAlert={createAlert}/>
+         </div>
+      );
+   }
```

Comment:
-0 Missing semi-colon.

While JS inserts semi-colons for you, it should not be relied upon and it is a best practice to always include them yourself.